src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -0,0 +1,40 @@
+package com.game.Battleship.models;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Board {
+
+    private int defaultBoardSize = 10;
```

Comment:
❓ Does this mean that you can override the default and make a bigger board later?

Reply:
yes

Reply:
Currently, how does one override `defaultBoardSize`? 🤔

Reply:
Nevermind, I didn't see the setter function. Do you anticipate that `Board` objects will change their sizes _after_ being initialized?

Reply:
Not after being initialized. But what I wanted to to is to be able to create a new instance with or without a size.When I create a new object and if I don't specify the size then the size attribute of the new object will 10(default size) or I can instantiate an object Board with a different size like: new Board(int 12). I modified the code by adding a new constructor.